MailCore/Models/Draft.swift

```diff
@@ -290,12 +290,14 @@ public struct UnmanagedDraft: Equatable, Encodable, AbstractDraft {
         }
         from = [Recipient(email: signature.sender, name: signature.fullName)]
         replyTo = [Recipient(email: signature.replyTo, name: "")]
-        let html = "<div><br></div><div><br></div><div class=\"editorUserSignature\">\(signature.content)</div>"
-        switch signature.position {
-        case .top:
-            body.insert(contentsOf: html, at: body.startIndex)
-        case .bottom:
-            body.append(contentsOf: html)
+        if !body.contains("editorUserSignature") {
+            let html = "<div><br></div><div><br></div><div class=\"editorUserSignature\">\(signature.content)</div>"
```

Comment:
```suggestion
            let html = "<br><br><div class=\"editorUserSignature\">\(signature.content)</div>"
```
You can do the same on line 258